collections/_posts/2022-06-13-Physics-InformedNeuralNetworks-espresso.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: review
+title: "Flow over an espresso cup: Inferring 3D velocity and pressure fields from tomographic background oriented schlieren videos via physics-informed neural networks"
+tags: Flow Physics-Informed-NN PINN
```

Comment:
I propose you to put more standard tag, for instance replace NN by neural network. Here is a suggestion: 
flow, physics, neural network

Reply:
Changed !